Sources/PrimerSDK/Classes/Core/Primer/Primer.swift

```diff
@@ -165,6 +166,7 @@ public class Primer {
                     .adyenTwint,
                     .adyenVipps,
                     .adyenPayshop,
+                    .applePay,
```

Comment:
do we need `.applePay` among these payment method cases?

Reply:
Yes, apple pay is only for checking out